apps/frontend/src/pages/profile/overview.tsx

```diff
@@ -62,11 +62,11 @@ export default function ProfileOverview() {
     >
       <div className="flex flex-col gap-3 mt-[46px]">
         <div className="flex flex-col px-4">
-          <span className="text-[30px] font-semibold tracking-[-0.22px] font-sans text-label-primary">
-            {user?.userData.displayName}
+          <span className="text-[30px] font-semibold tracking-[-0.22px] font-sans">
+            {`${user?.userData.username}`}
```

Comment:
In one location we had username at the top and display below it, and two cases with the reverse. Given that quite a few users have not filled out the display name, and username is required, I swapped the naming to consistently put username first and display name below it.